edb/lib/std/25-numoperators.edgeql

```diff
@@ -2094,7 +2095,8 @@ std::`//` (n: std::int16, d: std::int16) -> std::int16
     # the common attributes of the SQL division operator while
     # overriding the main operator function.
     USING SQL OPERATOR r'/';
-    USING SQL 'SELECT floor("n"::numeric / "d"::numeric)::int2';
+    USING SQL
+        'SELECT floor(1.0::numeric * "n"::numeric / "d"::numeric)::int2';
```

Comment:
Now that we have fast and working `%`, should we use it here instead of `floor`?

Reply:
We would need a way of handling the case when `d` is close to minimum integer value and subtracting the remainder would push it beyond the valid range. Similar issue as for `%`, but a bit harder to actually tackle, because we're supposed to unconditionally subtract the remainder.